.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 dist: trusty
 language: cpp
-compiler: g++
+compiler: clang
```

Comment:
just out of curiosity: the new image doesn't have gcc7 by default? telling as i think gcc is currently faster than clang